test-e2e/app/Tests/MatrixPage/MatrixPageFilters.e2e-spec.ts

```diff
@@ -1,7 +1,7 @@
-import { browser } from 'protractor';
+import {browser} from 'protractor';
```

Comment:
WEbstormSettings => Editor => CodeStyle => TypeScript => Spaces => ES^ import/export braces - should be checked

Reply:
Done.